src/Vlc.DotNet.Core.Interops/Vlc.DotNet.Core.Interops.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net20;net35;net40;net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net20;net35;net40;net45;netstandard2.0;netstandard1.3</TargetFrameworks>
```

Comment:
Do we need to keep both netstandard2.0 and netstandard1.3 ?

Reply:
I would suggest to do so; some code is missing in netstandard1.3 (like `Encoding.Default`) which is present in `netstandard2.0`. So the netstandard1.3 & 2.0 builds are not exactly the same.

Which brings me to another question - why do target .NET 2.0, 3.5, 4.0 and 4.5?

Reply:
Backward compatibility I guess. Not sure if the original developer knows himself if this is still needed...